### PR TITLE
Fix keyinputs

### PR DESCRIPTION
**Pull request by macocha**
This changes two things:
 - It explicitly sends shift off key before sending a char which doesn't need it. Which hopefully will fix the weirdness some folks were getting.
 - Other thing is that it locks down all other inputs while it's pasting the text, so it won't be disturbed. I had to unlock a little earlier than the last char is pasted, because it would get unresponsive when normal typing. ;) Actually, I think I have an idea how to fix that. brb. It's me again, now we're locking input only when we are really pasting something, not when just typing. 🎉

### CODE REVIEW

**Comment by macocha** [comment]
@jbarber

File: Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as [hunk 1]
```diff
@@ -142,6 +142,15 @@ package com.flashlight.vnc
 			}
 		}
 
+		private static function jsLog(message:String):void {
+		  if (ExternalInterface.available) {
+		    try {
+		      ExternalInterface.call("flashLog", message);
+		    } catch (e:Error) {
```
Comment (jbarber) [inline Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as]: As a [wise man once said to me](https://github.com/rainforestapp/Flashlight-vnc/pull/21#issuecomment-197282409):
> silently catching errors can get tricky down the road

LOL

Comment (macocha) [inline Flashlight-VNC/src/com/flashlight/vnc/VNCClient.as]: :trollface:

**Comment by macocha** [comment]
I'll reopen as a merge from develop

**Pull request closed without merging**